### PR TITLE
Update template model

### PR DESCRIPTION
**Pull request by JohnL17**

### CODE REVIEW

**Review by open-dynaMIX — APPROVED** [review]
Looks good :tada:. But I was the navigator ;) So let's also wait for a review by @luytena .

File: document_merge_service/api/serializers.py
Comment (luytena) [inline document_merge_service/api/serializers.py:75]: @open-dynaMIX Thanks for the pair implementation with @JohnL17! Quick question: I didn't quite understand which group is written into the created_by_group / modified_by_group fields, or better said where the data is coming from and how it relates to the user's group in eBau. Could you provide some insights? :)

Comment (open-dynaMIX) [reply document_merge_service/api/serializers.py:75]: This comes from the IDP. We get a list of groups (strings), [accessible via `user.groups`](https://github.com/adfinis/document-merge-service/blob/main/document_merge_service/api/authentication.py#L34). We treat the first group as main group and [make it accessible via `user.group`](https://github.com/adfinis/document-merge-service/blob/main/document_merge_service/api/authentication.py#L18).

Comment (luytena) [reply document_merge_service/api/serializers.py:75]: Thanks for clarifying :+1:

**Review by luytena — APPROVED** [review]
Looks good to me :100:

**Pull request merged**
